core/trino-main/src/main/java/io/trino/sql/planner/iterative/rule/PushLimitIntoTableScan.java

```diff
@@ -37,6 +38,7 @@
     private static final Capture<TableScanNode> TABLE_SCAN = newCapture();
     private static final Pattern<LimitNode> PATTERN = limit()
             .matching(limit -> !limit.isWithTies())
+            .with(isOrderSensitive().equalTo(false))
```

Comment:
Add a comment explaning that it's because this concept doesn't exist in the connector limit pushdown API